pydantic_core/core_schema.py

```diff
@@ -1430,9 +1433,7 @@ def dict_schema(
 
 
 class ValidatorFunction(Protocol):
-    def __call__(
-        self, __input_value: Any, *, data: Any, config: CoreConfig | None, context: Any, **future_kwargs: Any
-    ) -> Any:  # pragma: no cover
+    def __call__(self, __input_value: Any, info: ValidatorInfo) -> Any:  # pragma: no cover
```

Comment:
technically the signature here should be 
```suggestion
    def __call__(self, __input_value: Any, __info: ValidatorInfo) -> Any:  # pragma: no cover
```

since it's a positional only argument, we should include the double underscore, also good to be consistent with `__input_value`.

Reply:
changes needs to be duplicated to other places in this file.

Reply:
done